simulator/src/main/java/simblock/task/WinningTask.java

```diff
@@ -0,0 +1,29 @@
+package simblock.task;
+
+import simblock.node.Node;
+import simblock.simulator.Simulator;
+import simblock.simulator.Timer;
+
+public class WinningTask implements Task{
+
+    public static int k = 0;
+
+    public long getInterval(){
+        return 10 * (++k);
+    }
+
+    /**
+     * Run the task.
+     */
+    public void run(){
```

Comment:
When the task is run by the simulator, it determines who's turn it is by getting the next ready node, then adds that nodes minting task to the simulator 1 time unit later